metrics/web/test/common/presentation/navigation/metrics_route_information_parser_test.dart

```diff
@@ -87,14 +97,26 @@ void main() {
     );
 
     test(
-      ".restoreRouteInformation() returns the route information with the location equals to the given route configuration path",
+      ".restoreRouteInformation() delegates converting the route configuration to a location to the route configuration location converter if it is not null",
```

Comment:
🏷️
```suggestion
      ".restoreRouteInformation() delegates converting the route configuration to the route configuration location converter if the configuration is not null",
```
Would it be a bit more clear?